modules/neo4j/src/main/java/org/testcontainers/containers/Neo4jContainer.java

```diff
@@ -60,6 +63,8 @@
 
     private String adminPassword = DEFAULT_ADMIN_PASSWORD;
 
+    private final Set<Neo4jLabsPlugin> labsPlugins = new HashSet<>();
```

Comment:
Let's store this as `Set<String>` and add an overloaded `withLabsPlugins(String...)`, so that we don't *have* to release a new version of TC when you folks add a new plugin to your images 👍

Reply:
I am resolving this comment since it was changed as proposed.